build/common.project.props

```diff
@@ -15,7 +15,8 @@
     <NETFXTargetFramework>net472</NETFXTargetFramework>
     <NETCoreTargetFramework>netcoreapp3.1</NETCoreTargetFramework>
     <NETCoreTargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</NETCoreTargetFramework>
-    <NETCoreTestTargetFrameworks>net7.0;net8.0</NETCoreTestTargetFrameworks>
+    <NETCoreTestTargetFrameworks Condition=" '$(NUGET_CLIENT_VS_INSTALLED_TOOLS_ONLY)' != 'true' ">net7.0;net8.0</NETCoreTestTargetFrameworks>
+    <NETCoreTestTargetFrameworks Condition=" '$(NUGET_CLIENT_VS_INSTALLED_TOOLS_ONLY)' == 'true' ">net7.0</NETCoreTestTargetFrameworks>
```

Comment:
I'm open to suggestions for better environment variable names.  `NUGET_CLIENT_VS_INSTALLED_TOOLS_ONLY` isn't great. It's not even good. But I thought it should contain the name of our repo to reduce risk we use a name that conflicts with other software, and then the rest of my proposed name is descriptive 🤷

Reply:
Hopefully its just temporary so the name doesn't matter.